projects/lib/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "ngrx-store-localstorage",
+  "version": "9.0.0",
+  "peerDependencies": {
+    "@angular/common": "^12.0.0",
+    "@angular/core": "^12.0.0",
+    "@ngrx/store": "^12.0.0"
```

Comment:
Previously this was quite flexible 

```json
"@ngrx/store": "^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0"
```

Will this only work with version 12 onwards? Or was there a specific reason you made this explicitly for version 12?

Reply:
@BBlackwo in theory, it should work, but I haven't tested the library-generated project on older versions of ngrx/store and angular, only on ng12, hence the explicit version 12.

Don't really have a strong opinion either way. I can remove the angular peer dependency, and set the store dependency back to how it was, your call.

The rationale was that people most likely would only be upgrading this lib when the upgrade to ng12.

Reply:
Yeah let's go with what you've done for now. People will most likely be keeping up to date with Angular anyways.